manifests/m/Microsoft/AzureCLI/2.42.0/Microsoft.AzureCLI.yaml

```
@@ -0,0 +1,16 @@
+PackageIdentifier: Microsoft.AzureCLI
+PackageVersion: 2.42.0
+PackageName: Microsoft Azure CLI
+Publisher: Microsoft Corporation
+Moniker: azure-cli
+PackageUrl: https://docs.microsoft.com/en-us/cli/azure/
+License: Copyright (C) Microsoft Corporation
```

A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate 
```suggestion
License: Proprietary
Copyright: Copyright (C) Microsoft Corporation
```